### PR TITLE
Create scaleway-cli formula

### PR DESCRIPTION
**Pull request by nicolas-brousse**
I create a new formula to add the [scaleway-cli](https://github.com/scaleway/scaleway-cli).

It exists a binary for 32-bits OSs but I don't know how I have to precise it. Maybe it could be added.
And also I'm not sure if I have to precise bottle and how.

### CODE REVIEW

File: Library/Formula/scaleway-cli.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class ScalewayCli < Formula
+  desc "Interact with Scaleway API from the command-line"
+  homepage "https://github.com/scaleway/scaleway-cli/"
+  url "https://github.com/scaleway/scaleway-cli/releases/download/v1.1.0/scw-Darwin-x86_64"
+  version "1.1.0"
+  sha256 "78ac104bf11f856ea50e7c75442264e5601214b5d33118648866250fcc0bff80"
+
+  def install
+    mv "scw-Darwin-x86_64", "scw"
+    bin.install "scw"
```
Comment (bfontaine) [inline Library/Formula/scaleway-cli.rb]: You can do that in one line:

```
bin.install "scw-Darwin-x86_64" => "scw"
```

Comment (nicolas-brousse) [inline Library/Formula/scaleway-cli.rb]: Thanks for the tip

**Comment by DomT4** [comment]
It looks like this is possible to build from source. Can we do that instead? Thanks!

**Comment by nicolas-brousse** [comment]
I will check and may be ask them :)

**Comment by DomT4** [comment]
[This](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/gauge.rb) is a good example of building a `Go` tool in Homebrew from source if you need it :)

**Comment by nicolas-brousse** [comment]
Thank you. I will take a look tomorrow.

**Comment by nicolas-brousse** [comment]
Well... I just see that they create a formula into there repository: https://github.com/scaleway/scaleway-cli/blob/master/contrib/builder/homebrew/scw.rb. :(

**Comment by DomT4** [comment]
It could be worth asking upstream if they'd be happy for you to submit a formula (built from source but with Homebrew's binaries available to end-users) here to replace their's.

**Comment by mikemcquaid** [comment]
> It could be worth asking upstream if they'd be happy for you to submit a formula (built from source but with Homebrew's binaries available to end-users) here to replace their's.

:+1:

**Comment by nicolas-brousse** [comment]
I have no answers from them until now. I will keep you in informed.

**Comment by moul** [comment]
We will release our formula soon, we need to test a little bit more the shell autocompletion scripts and bump commit ids

**Comment by brmzkw** [comment]
As I told to @nicolas-brousse in private, @moul intends to submit a formulae but he needs to work a bit more on it before.

**Comment by nicolas-brousse** [comment]
So I close my PR

**Pull request closed without merging**